airflow/api_connexion/schemas/task_instance_schema.py

```diff
@@ -55,6 +55,7 @@ class Meta:
     state = TaskInstanceStateField()
     _try_number = auto_field(data_key="try_number")
     max_tries = auto_field()
+    task_display_name = fields.String(attribute="task_display_name")
```

Comment:
```suggestion
    task_display_name = fields.String(attribute="task_display_name", dump_only=True)
```

?

Reply:
Okay, I ledt this like the others in dag_schema.py. Seems to be not 100% conststent with the schemas in general. In task_instance.py all fields are dump-only, in dag_schema they are not.
Anyway, fixed it, might be a consistency PR following...